framework/doc/content/source/meshgenerators/HexToTetMeshGenerator.md

```diff
@@ -0,0 +1,14 @@
+# HexToTetMeshGenerator
```

Comment:
thoughts about ElementTypeConversionGenerator?

If we dont do that we may end up with like 12 or more generators for conversion (6 from tri / quad / poly (one day), dozens from tet / hex / prism / pyramid / poly).
One argument is that we dont expect to see more than one of these per input